resources/text_parser_realized.json

```diff
@@ -0,0 +1 @@
+{"foreign_key_config": {"field_name": "patent_id", "friendly_name": "Patent Number", "xml_path": "us-bibliographic-data-grant/publication-reference/document-id/doc-number", "data-type": "string"}, "table_xml_map": [{"table_prefix": "claims", "friendly_name": "Claim", "multi_valued": true, "entity_root_path": "claims/", "fields": [{"field_name": "text", "friendly_name": "Text", "xml_path": "claim-text", "attribute": null, "description": null, "sequence": false, "flag": null, "tag": false, "data-type": "fulltext"}, {"field_name": "sequence", "friendly_name": "Sequence", "xml_path": ".", "attribute": null, "description": null, "sequence": true, "flag": null, "tag": false, "data-type": "string"}, {"field_name": "num", "friendly_name": "num", "xml_path": ".", "attribute": "num", "description": null, "sequence": false, "flag": null, "tag": false, "data-type": "string"}, {"field_name": "dependent", "friendly_name": "Dependent", "xml_path": "./claim-text/claim-ref", "attribute": null, "description": null, "sequence": false, "flag": null, "tag": false, "data-type": "string"}], "filename": "claim.tsv", "table_name": "claims_2021"}, {"table_prefix": "brf_sum_text", "friendly_name": "Brief Summary Text", "multi_valued": false, "fields": [{"field_name": "text", "friendly_name": "Text", "xml_path": "description", "description": "Brief Summary", "attribute": null, "sequence": false, "flag": null, "tag": false, "data-type": "fulltext"}], "filename": "brf_sum_text.tsv", "table_name": "brf_sum_text_2021"}, {"table_prefix": "claim_exemplary", "friendly_name": "Claim Exemplary", "multi_valued": false, "fields": [{"field_name": "exemplary", "friendly_name": "Exemplary Claim", "xml_path": "us-bibliographic-data-grant/us-exemplary-claim", "description": null, "attribute": null, "sequence": false, "flag": null, "tag": false, "data-type": "string"}], "filename": "exemplary_claim.tsv", "table_name": "claim_exemplary_2021"}, {"table_prefix": "draw_desc_text", "friendly_name": "Drawing Description Text", "multi_valued": true, "entity_root_path": "description/description-of-drawings/", "fields": [{"field_name": "text", "friendly_name": "Text", "xml_path": ".", "description": "Brief Description of Drawings", "sequence": false, "attribute": null, "flag": null, "tag": false, "data-type": "fulltext"}, {"field_name": "sequence", "friendly_name": "Sequence", "xml_path": ".", "attribute": null, "description": null, "sequence": true, "flag": null, "tag": false, "data-type": "string"}], "filename": "draw_desc_text.tsv", "table_name": "draw_desc_text_2021"}, {"table_prefix": "detail_desc_text", "friendly_name": "Detailed Description Text", "multi_valued": false, "fields": [{"field_name": "text", "friendly_name": "Text", "xml_path": "description", "description": "Detailed Description", "sequence": false, "attribute": null, "flag": null, "tag": false, "data-type": "fulltext"}], "filename": "detail_desc_text.tsv", "table_name": "detail_desc_text_2021"}]}
```

Comment:
The `text_parser_realized.json` is best left untracked by git. This is a dynamically created file that will change every time the TextTest runs creating git status change.